xmake.lua

```diff
@@ -1,15 +1,15 @@
 add_rules("mode.debug", "mode.release")
 
-add_requires("asio", "nlohmann_json")
+add_requires("asio", "nlohmann_json","spdlog","gtest")
 add_requires("range-v3")
 set_languages("cxxlatest")
 -- set_optimize("aggressive")
 set_optimize("fastest")
-set_warnings("more", "error")
+-- set_warnings("more", "error")
 
 target("nogo")
     set_kind("binary")
-    add_packages("asio", "nlohmann_json")
+    add_packages("asio", "nlohmann_json","spdlog","gtest")
```

Comment:
Is the `gtest` here necessary? In my opinion, testing should be another target.

Reply:
I didn't delete it when I pushed last time. Wait a minute.